polaris-tokens/src/themes/light-mobile.ts

```diff
@@ -0,0 +1,63 @@
+import {createVar} from '../utils';
+
+import {createMetaTheme, createMetaThemePartial} from './utils';
+
+export const metaThemeLightMobilePartial = createMetaThemePartial({
+  text: {
+    // heading-2xl
+    'text-heading-2xl-font-size': {
+      value: createVar('font-size-800'),
+    },
+    // heading-xl
+    'text-heading-xl-font-size': {
+      value: createVar('font-size-550'),
+    },
+    'text-heading-xl-font-line-height': {
```

Comment:
~~Is there a reason the line-heights are defined for most of the styles here, but not for `heading-2xl`, `heading-lg`, `heading-md` and `heading-sm`?~~

Oh I think it's because the values are the same on desktop.

Reply:
Good question! This new `light-mobile` theme only contains Mobile typography overrides of the base theme (ensuring types are consistent between base/variant themes and helps keep our stylesheet as small as possible e.g.

```css
:root, :root:.p-theme-light {
  /* All base theme custom properties */
}
:root.p-theme-light-high-contrast {
  /* Only base theme custom property overrides */
}
:root.p-theme-light-mobile {
  /* Only base theme custom property overrides */
}
```